src/components/oppsummering/vilkårsOppsummering/faktablokk/faktablokker/UførhetFaktablokk.tsx

```diff
@@ -58,22 +66,28 @@ export const UførhetVilkårsblokk = (props: VilkårsblokkProps<'uførhet'>) =>
                             {
                                 tittel: saksbehandlingMessage('radio.uførhet.legend'),
                                 verdi:
-                                    props.behandlingsinformasjon.status === UførhetStatus.VilkårOppfylt
+                                    props.grunnlagsdataOgVilkårsvurderinger.uføre.resultat ===
+                                    UføreResultat.VilkårOppfylt
                                         ? intl.formatMessage({ id: 'fraSøknad.ja' })
-                                        : props.behandlingsinformasjon.status === UførhetStatus.VilkårIkkeOppfylt
+                                        : props.grunnlagsdataOgVilkårsvurderinger.uføre.resultat ===
+                                          UføreResultat.VilkårIkkeOppfylt
                                         ? intl.formatMessage({ id: 'fraSøknad.nei' })
                                         : intl.formatMessage({ id: 'radio.label.uføresakTilBehandling' }),
                             },
-                            ...(props.behandlingsinformasjon.status === UførhetStatus.VilkårOppfylt
+                            ...(props.grunnlagsdataOgVilkårsvurderinger.uføre.resultat === UføreResultat.VilkårOppfylt
                                 ? [
                                       FaktaSpacing,
                                       {
                                           tittel: saksbehandlingMessage('input.label.uføregrad'),
-                                          verdi: props.behandlingsinformasjon.uføregrad?.toString() ?? '-',
+                                          verdi:
+                                              props.grunnlagsdataOgVilkårsvurderinger.uføre.vurderinger[0].grunnlag?.uføregrad?.toString() ??
+                                              '-',
                                       },
                                       {
                                           tittel: saksbehandlingMessage('input.label.forventetInntekt'),
-                                          verdi: props.behandlingsinformasjon.forventetInntekt?.toString() ?? '-',
+                                          verdi:
+                                              props.grunnlagsdataOgVilkårsvurderinger.uføre.vurderinger[0].grunnlag?.forventetInntekt?.toString() ??
+                                              '-',
```

Comment:
er väl ikke riktig å kun vise `...vurderinger[0].grunnlag?.....`, den kan väl finnes flere grunnlag som ligger inne? isåfall ønsker man ju å vise alla de grunnlagen. 😊

Reply:
Eller hmm, kanske er sån att den må være for hele perioden i søknadsbehandlingen så kanske stemmer likevel. 😅

Reply:
Er lagt opp i backend at man i _fremtiden_ skal kunne støtte flere vurderinger/perioder med uføre. Enn så lenge gjøres ikke det, og derfor vil det alltid være enten `null`, tom liste, eller en liste med bare ett innslag:)

Reply:
hehe yes, misstänkte det. du kan ju vurdere å lage en funksjon `getDaFirstVurdering(x)` som gjør akkurat det där, sån att når vi får støtte for flere perioder så kan vi bara slette den funksjonen och så finner vi de stederne det må fikses. 😄

Reply:
Begynte å lage det, men var så få steder at det er like mye å huske på den funksjonen 😅 Men trakk ut uføre som en variabel, så ble det litt penere 😊